test/topology_custom/test_wait_for_ip_persistence.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import logging
+import asyncio
+import pytest
+from test.pylib.util import wait_for_cql_and_get_hosts, wait_for
+from test.pylib.rest_client import inject_error_one_shot
+import time
+from test.pylib.manager_client import ManagerClient
+from test.topology.conftest import skip_mode
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_wait_for_ip_persistence(manager: ManagerClient):
+    logger.info("starting initial cluster with two nodes")
+    s1 = await manager.server_add()
+    s2 = await manager.server_add()
```

Comment:
You can boot them concurrently with `servers_add`, may be a little bit faster

Reply:
I want to boot `s1` first to be sure it's a leader.